cmp/form/field/BaseFieldModel.js

```diff
@@ -162,9 +171,9 @@ export class BaseFieldModel {
 
         // Force an immediate 'Unknown' state -- the async recompute leaves the old state in place until it completed.
         // (We want that for a value change, but not reset/init)  Force the recompute only if needed.
```

Comment:
This comment is a bit difficult to grok - I admittedly can't follow what it's saying.